plugin/src/main/java/io/jenkins/plugins/analysis/core/portlets/PullRequestMonitoringPortlet.java

```diff
@@ -96,13 +97,43 @@ public String getResultIssuesJsonModel() {
     }
 
     /**
-     * Check if {@link AnalysisResult} has issues.
+     * Check if {@link AnalysisResult} issues are empty.
      *
      * @return
-     *          true if {@link AnalysisResult} has issues, else false.
+     *          true if {@link AnalysisResult} issues are empty, else false.
      */
-    public boolean hasIssues() {
-        return action.getResult().hasIssues();
+    public boolean isEmpty() {
+        return action.getResult().isEmpty();
+    }
+
+    /**
+     * Check if action has a quality gate.
+     *
+     * @return
+     *          true if action has a quality gate, else false.
+     */
+    public boolean hasQualityGate() {
+        return !action.getResult().getQualityGateStatus().equals(QualityGateStatus.INACTIVE);
+    }
+
+    /**
+     * Get the icon of the quality gate.
+     *
+     * @return
+     *          the image url of the icon.
+     */
+    public String getQualityGateResultIconUrl() {
+        return action.getResult().getQualityGateStatus().getResult().color.getImageOf("16x16");
```

Comment:
This is still a bug in my plugin. I need to replace these icons with the new Jenkins SVG status images.

Reply:
So I can leave it like this for now?

Reply:
Yes, nothing to do on your side. I just noticed that I need to fix it in the plugin (or in Jenkins core), I just forgot about it...